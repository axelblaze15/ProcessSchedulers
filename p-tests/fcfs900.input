1000000000000000000 2 1
10000000000000 2 1
1000000000 2 1
1000000 2 1
1000 2 1
10 5 1