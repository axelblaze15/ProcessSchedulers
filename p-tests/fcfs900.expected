10 8 2 3
1000 3 0 1
1000000 4 1 2
1000000000 5 2 3
10000000000000 6 3 4
1000000000000000000 7 4 5
2 3